src/Controller/ContactFormController.php

```diff
@@ -72,6 +79,18 @@ public function createContactMessage(Request $request): Response
     {
         $contact = new ContactFormMessage();
         $sitekey = $_ENV['RECAPTCHA_SITE_KEY'];
```

Comment:
Resolve ENV variable in services.yml and hand over class constructor

Reply:
https://github.com/mangoweb-sylius/SyliusContactFormPlugin/pull/4#discussion_r358761711

Reply:
next issue (https://mangoweb.org/apps/sylius-test-plugin/issues/20)

Reply:
https://github.com/mangoweb-sylius/SyliusContactFormPlugin/pull/6

Reply:
what does it mean?

Reply:
already resolve